src/topp/FileMerger.cpp

```diff
@@ -204,6 +257,12 @@ class TOPPFileMerger :
             it->setMetaValue("file_origin", DataValue(file_list[i]));
           }
         }
+
+        if (rt_gap_ > 0.0) // concatenate in RT
+        {
+          adjustRetentionTimes_(map, trafo_out[i], i == 0);
```

Comment:
I think it's sufficiently clear. "i == 0" was already used to test for the first file elsewhere in the FileMerger code (now line 417).

That said, I can add a bool variable if you really want.